controllers/operator/create/create.go

```diff
@@ -123,25 +123,26 @@ func HandlePVCResize(ctx context.Context, memberClient kubernetesClient.Client,
 	// and we are not in the middle of a resize (that means pvcPhase is pvc.PhaseNoAction) for this statefulset.
 	// This means we want to start one
 	if increaseStorageOfAtLeastOnePVC {
-		err := enterprisests.AddPVCAnnotation(desiredSts)
-		if err != nil {
+		if err := enterprisests.AddPVCAnnotation(desiredSts); err != nil {
 			return workflow.Failed(xerrors.Errorf("can't add pvc annotation, err: %s", err))
 		}
+
 		log.Infof("Detected PVC size expansion; patching all pvcs and increasing the size for sts: %s", desiredSts.Name)
-		if err := resizePVCsStorage(memberClient, desiredSts); err != nil {
+		if err := resizePVCsStorage(ctx, memberClient, desiredSts, log); err != nil {
```

Comment:
`resizePVCsStorage()` function did not receive parent context, this is fixed now